CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+v0.4.9 / 2020-04-15
+========================
+
+  * add physical/logical block size, hardware sector size and drive type into BlockDevice resource
+  ([#388](https://github.com/openebs/node-disk-manager/pull/388), [@akhilerm](https://github.com/akhilerm))
+  * add new sysfs probe to fetch block device details from sysfs ([#375](https://github.com/openebs/node-disk-manager/pull/375), 
+  [shovanmaity](https://github.com/shovanmaity))
+  * enable persisting of labels and annotations on BlockDevice resource ([#394](https://github.com/openebs/node-disk-manager/pull/394), 
+  [shovanmaity](https://github.com/shovanmaity))
+  * add label selector to BlockDeviceClaim resource ([#397](https://github.com/openebs/node-disk-manager/pull/397), 
+  [@akhilerm](https://github.com/akhilerm))
+  * add support for `openebs.io/block-device-tag` label on BlockDevice ([#400](https://github.com/openebs/node-disk-manager/pull/400), 
+  [@akhilerm](https://github.com/akhilerm))
+
+v0.4.9-RC2 / 2020-04-11
+========================
+  * No changes since RC1
```

Comment:
@kmova Is this actually required?. IMO, we should not keep a section just for the title.

Reply:
Dont' have a strong opinion on this one. Remove the pr/hold-merge, after you have resolved this comment with either removing it or keep it. Either way is fine for me.

Reply:
Removed the RC2 title.